pkgs/by-name/z3/z3-cmake/package.nix

```diff
@@ -0,0 +1,18 @@
+{stdenv, z3, cmake, python3}:
+
+stdenv.mkDerivation {
+  inherit (z3) pname version src;
```

Comment:
Not sure how to handle this better. This package is temporary, and can be removed after https://github.com/NixOS/nixpkgs/issues/105358

However, it is not possible to implement current z3 package using cmake, because of ocaml bindings: https://github.com/Z3Prover/z3/pull/7254

Perhaps it would be better to move this package to the same file as prusa-slicer, as the only user of this package at this moment, but it feels wrong to mix slicer and build system in the same file.